tests/src/Functional/OnOffSwitchTest.php

```diff
@@ -24,7 +24,7 @@ class OnOffSwitchTest extends BrowserTestBase {
   /**
    * Data provider for testOnOffSwitch().
    */
-  public function onOffProvider() {
+  public static function onOffProvider() {
```

Comment:
Nit pick. If we're adding return types this should be `: \Generator` ?

Reply:
Good flag. Not sure why rector didn't flag this but I've updated the PR